### PR TITLE
ci(azure pipeline): remove Ansible 2.11 from Azure pipeline

### PR DESCRIPTION
**Pull request by hunleyd**
remove Ansible 2.11 from Azure pipeline

### CODE REVIEW

**Comment by Andersson007** [comment]
We should figure out what happened to the `postgresql_replication` module on Ubuntu 20.04.
The primary and replica can't start. Strange that it worked in the past.
@hunleyd do you have time to take a look? (no rush)

**Comment by hunleyd** [comment]
yeah, i have no clue how to fix this @Andersson007

**Comment by hunleyd** [comment]
for some reason, the ubuntu  boxes have both 14 and 15 installed, so the `find` is finding multiple `initdb` and multiple `pg_ctl` @Andersson007

**Comment by hunleyd** [comment]
even after fixing the code to deal w/ finding multiple PG binaries, the cluster still won't start and I can't figure out why. I think I don't understand the flow of the tests well enough to debug further @Andersson007

**Comment by Andersson007** [comment]
> even after fixing the code to deal w/ finding multiple PG binaries, the cluster still won't start and I can't figure out why. I think I don't understand the flow of the tests well enough to debug further @Andersson007

@hunleyd I'll take a look next week, np!

**Comment by Andersson007** [comment]
@hunleyd hm, the tests with `--docker ubuntu2004` work (and pass) locally...

File: tests/integration/targets/postgresql_subscription/defaults/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 pg_user: postgres
 db_default: postgres
-primary_port: 5433
+primary_port: 5431
```
Comment (Andersson007) [inline tests/integration/targets/postgresql_subscription/defaults/main.yml:3]:
```suggestion
primary_port: 5431
```
if it works, we could try to remove it from here as it's already defined in `tests/integration/targets/setup_postgresql_replication/defaults/main.yml` (i'm not 100% sure but it should work w/o having it here as well as in the same file of postgresql_info target)

Comment (Andersson007) [reply tests/integration/targets/postgresql_subscription/defaults/main.yml:3]: no need to remove it now but we could try at the end, after CI is green

**Comment by hunleyd** [comment]
how does `postgresql-14` work in your pr but not mine @Andersson007 ?

**Comment by Andersson007** [comment]
> how does `postgresql-14` work in your pr but not mine @Andersson007 ?

@hunleyd in the AZP log, it shows `"No package matching 'apt-utils - postgresql-14' is available"`, i.e. that the apt module can't install  the package with the name `apt-utils - postgresql-14`, which is expected, I think:)
I can see a tiny difference in the `pg_package_list` lists from our files:) PTAL

File: tests/integration/targets/postgresql_subscription/defaults/main.yml (hunk 1 above)
Comment (hunleyd) [reply tests/integration/targets/postgresql_subscription/defaults/main.yml:3]: ok, trying this now

**Review by jchancojr — APPROVED** [review]
Interesting find! lgtm

**Pull request merged**